src/btree/row_srch.c

```diff
@@ -785,7 +785,7 @@ __wt_row_random_leaf(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt)
  *	Find a random leaf page in a row-store tree.
  */
 int
-__wt_row_random_descent(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt)
```

Comment:
I agree with this, and I've pushed that change in 7666562.

I left the function itself in `row_srch.c` because I can't think of a better place for it.